.gitpod.yml

```diff
@@ -0,0 +1,15 @@
+tasks:
```

Comment:
Is it okay if I add a `.gitpod.yml` file to make the development in [gitpod.io](https://gitpod.io) (which I use) a bit more pleasant?

Reply:
Yeah, looks good. Thanks for adding this.